vendor/github.com/Dynatrace/libbuildpack-dynatrace/hook.go

```diff
@@ -228,18 +237,22 @@ func (h *Hook) getCredentials() *credentials {
 }
 
 // download gets url, and stores it as filePath, retrying a few more times if the downloads fail.
-func (h *Hook) download(url, filePath string) error {
+func (h *Hook) download(url, filePath string, buildPackVersion string, language string) error {
 	const baseWaitTime = 3 * time.Second
 
+	client := &http.Client{}
+	req, _ := http.NewRequest("GET", url, nil)
+	req.Header.Set("User-Agent", fmt.Sprintf("cf-%s-buildpack/%s", language, buildPackVersion))
```

Comment:
Why do we need to include language and buildpack version in the header now?

Reply:
Hey @ndon55555 !
Since our customers use a lot of different buildpacks and versions we would like to get a matrix of all the used versions and buildpacks to check which versions are in use and which aren't.
Hope this isn't a problem :)

Reply:
@ndon55555:
Is it okay to have it like that?
Any update on the status?

Reply:
Hey @ndon55555!
Any update on this?